Server/AuthService/schemas.py

```
@@ -0,0 +1,222 @@
+            "example": {
+                "email": "admin@faceofmind.it.com",
+                "password": "securepassword123"
+            }
+        }
+
+
+class TokenResponse(BaseModel):
+    """Token response schema."""
+    access_token: str
+    refresh_token: str
+    token_type: str = "bearer"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "token_type"

(S105)